docs/content/config/advanced/kubernetes.md

```diff
@@ -140,6 +140,7 @@ If using our Helm chart is not viable for you, here is some guidance to start wi
                 app: mailserver
 
             spec:
+              externalTrafficPolicy: Local # most likely required in all scenarii, else every incoming request would be identified as the External IP, which will get banned by Fail2Ban eventually 
```

Comment:
```suggestion
              externalTrafficPolicy: Local # most likely required in all scenarii, else every incoming request would be identified as the external IP address, which will get banned by Fail2Ban eventually 
```

Reply:
```suggestion
              # `Local` is most likely required, otherwise every incoming request would be identified by the external IP,
              # which will get banned by Fail2Ban when monitored services are not configured for PROXY protocol
              externalTrafficPolicy: Local
```

- Shift comment to above field, split to two lines.
- Drop `scenarii` (_that doesn't seem to be a common use for what I assume is plural equivalent of scenarios?_)
- Added additional context/hint for the concern being related to lack of PROXY protocol usage.

I am not familiar with k8s to review further as I don't know how `Local` fixes the concern here. Or if "external IP" is referring to ingress/lb or real client IP.

Since PROXY protocol is usually required for k8s from what @georglauterbach has said in the past IIRC, I assume `Local` doesn't solve the concern across the board with external traffic?🤷‍♂️